### DISCUSSION TITLE
Pull shark achievement not appeared

### DISCUSSION THREAD

**Discussion by kulikov-dev** [community/community#36174] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi there. I'm in doubts.
I have about 25 merged pull requests for my organization projects and a few to another one.
However I still didn't get this achievment. "show achievments option" is enabled.
Any ideas why and how to fix it?

**Comment by FahdSeddik** [comment, marked as answer]
Hey @kulikov-dev 
For pull shark to show up, at least 2 pull requests have to be **_successfully merged_**. Make sure you follow this [link](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/showing-your-private-contributions-and-achievements-on-your-profile#changing-the-visibility-of-achievements). Also, make sure private activity is enabled for your activity panel. Hope this helps!
![image](https://user-images.githubusercontent.com/62207434/195923327-e0580a6b-c401-400b-bc6c-f2f922d9b255.png)

**Reply by kulikov-dev** [reply]
Huh, private contributions made a trick. Thank you!
